doc/symfony-cache-configuration.rst

```diff
@@ -240,14 +245,17 @@ To get cache tagging support, register the ``PurgeTagsListener`` and use the
         SurrogateInterface $surrogate = null,
         array $options = []
     ) {
+        $tagsHeader = 'X-Cache-Tags';
```

Comment:
i think best practice would be to define this as a `const` on the class. or use `TagHeaderFormatter::DEFAULT_HEADER_NAME` but if this example wants to show how to change the header it makes sense to have our own constant